apps/web-giddh/webpack.test.partial.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
             'errlyticsNeeded': JSON.stringify(false),
             'errlyticsKey': JSON.stringify(''),
             'AppUrl': JSON.stringify('https://test.giddh.com/'),
-            'ApiUrl': JSON.stringify('https://apitest.giddh.com/'),
+            'ApiUrl': JSON.stringify('https://apirelease.giddh.com/'),
```

Comment:
it should be reverted i guess?

Reply:
we changed because apitest is not working and test is on hold